src/MLJGLMInterface.jl

```diff
@@ -45,24 +45,30 @@ const LCR_DESCR = "Linear count regressor with specified "*
 # LinearBinaryClassifier --> Probabilistic w Binary target // logit,cauchit,..
 # MulticlassClassifier   --> Probabilistic w Multiclass target
 
+const VALID_KEYS = [:deviance, :dof_residual, :stderror, :vcov, :coef_table]
+const DEFAULT_KEYS = VALID_KEYS # For more understandable warning mssg by `@mlj_model`.
+const KEYS_TYPE = Union{Nothing, AbstractVector{Symbol}}
 
 @mlj_model mutable struct LinearRegressor <: MMI.Probabilistic
     fit_intercept::Bool = true
     dropcollinear::Bool = false
     offsetcol::Union{Symbol, Nothing} = nothing
+    report_keys::KEYS_TYPE = DEFAULT_KEYS::(isnothing(_) || issubset(_, VALID_KEYS))
```

Comment:
Can this be written in another way? I have no idea what is happening here; or is this some kind of MLJ convention?

Reply:
Yeah, it's an MLJ language for adding constraints to model fields. I think this should have been documented in `@mlj_model` macro. But that's a issue for another day.